ethereum/p2p/src/main/java/org/hyperledger/besu/ethereum/p2p/discovery/internal/PingPacketData.java

```diff
@@ -77,6 +95,13 @@ public void writeTo(final RLPOutput out) {
         .encodeStandalone(out);
     to.encodeStandalone(out);
     out.writeLongScalar(expiration);
+    out.writeBytes(
+        getEnrSeq()
+            .orElseThrow(
+                () ->
+                    new IllegalStateException(
```

Comment:
How does Besu behave if this error is thrown ?

Reply:
The packet would not be created and an exception is thrown. I did it this way to follow the pattern done for missing from field.